features/steps/run.py

```
@@ -1,4 +1,4 @@
-from behave import when
+from behave import when, given
```

Ruff (0.8.2)

`behave.given` imported but unused

Remove unused import: `behave.given`

(F401)